src/pybind/mgr/dashboard/CMakeLists.txt

```diff
@@ -27,17 +27,20 @@ if(WITH_SYSTEM_NPM)
 else()
   set(mgr-dashboard-nodeenv-dir ${CMAKE_CURRENT_BINARY_DIR}/node-env)
   set(nodeenv NODEENV)
+  set(MGR-DASHBOARD-USERCONFIG --userconfig ${mgr-dashboard-nodeenv-dir}/.npmrc)
```

Comment:
nit, i'd suggest use lower-cased names for local variables.

Reply:
ack